### DISCUSSION TITLE
Media keys (play, prev., next) does not work in Audacious 4.2-1 (Debian 12 - stable package)

### DISCUSSION THREAD

**Discussion by slrslr** [audacious-media-player/audacious#1506] [Q&A]
By default my keyboard's playback keys does not work in Audacious 4.2-1 (Debian 12 + KDE - stable package).
It works in a Clementine player, maybe due to me using my system "Media Controller" Shortcuts to map these media keys to a Clementine. Doing the same for Audacious does not work. "Global hotkeys" plugin of Audacious does not recognize/map the key when pressed. Other, regular key shortcuts are mapped.

LANGUAGE=en_US:en
language map/layout is: cs/cz (non-english)
Model is: generic 101 key PC

After messing with the Keyboard shortcuts (system settings configuration) it started working. Possible steps inside Shortcuts configuration of my system:

- remove media keys assignment from other media players
- export working media app (in my case it was Clementine) scheme and adjust its copy to work for Audacious:
```
[audacious.desktop][Global Shortcuts]
next_album=Shift+Media Next
next_track=Media Next
play_pause=Media Play
prev_track=Media Previous
stop=Media Stop
```
and apply changes
- find Media controller, assign play key
- Restart Audacious
- try next/previous track media key, inside system notification popup (showing track details), click Play button.

then the play/pause media key started working in Audacious.

Created an issue for this: https://github.com/audacious-media-player/audacious/issues/1603

**Comment by pityalvarez** [comment]
Just fresh installed linux mint, installed audacious and have the same thing happen to me, so you're not alone.

**Comment by Flaconia** [comment]
This problem is still there in Audacious 4.4.2 working on Manjaro KDE.
Every other app, like Firefox, VLC or Elisa, that uses my Play/Pause Media button on my keyboard works without problems, always out of the box.



![immagine](https://github.com/user-attachments/assets/54153924-ac55-41fc-ac8c-9beaf9b89df9)

I've noticed that if I uncheck the "Play/Pause Media" box in KDE's hotkeys settings, Audacious actually recognises the key and I'm able to assign it in the Audacious Hotkey Plugin, but it obviously stops working in every other app which was relying on KDE's hotkey system.
With the hotkey back on in KDE, trying to make Audacious recognize the button to assign a different action in the plugin settings doesn't work, it simply doesn't auto-compile the input field, while it does with other non-media keys.

What I'd expect is Audacious to work like every other media playing app, to have the possibility to listen to KDE's hotkeys I guess which I'd suspect are some kind of universal key protocol?
